collect_app/src/test/java/org/odk/collect/android/activities/MainActivityTest.java

```diff
@@ -52,6 +64,36 @@ public void titleTest() throws Exception {
     }
 
     /**
+     * {@link Test} to assert Options Menu's functioning.
+     */
+    @Test
+    public void optionsMenuTest() throws Exception {
+        Menu menu = shadowOf(mainMenuActivity).getOptionsMenu();
+
+        assertNotNull(menu);
+        assertNotNull(mainMenuActivity.onCreateOptionsMenu(menu));
+
+        //Test for AboutPreferencesActivity
+        mainMenuActivity.onOptionsItemSelected(menu.getItem(0));
+        ShadowActivity shadowAboutPreferencesActivity = shadowOf(mainMenuActivity);
+        Intent startedIntentAbout = shadowAboutPreferencesActivity.getNextStartedActivity();
+        ShadowIntent shadowIntentAbout = shadowOf(startedIntentAbout);
+        assertEquals(AboutPreferencesActivity.class.getName(),shadowIntentAbout.getIntentClass().getName());
+
+        //Test for PreferencesActivity
+        mainMenuActivity.onOptionsItemSelected(menu.getItem(1));
+        ShadowActivity shadowGeneralPreferencesActivity = shadowOf(mainMenuActivity);
+        Intent startedIntentGeneralPreferences = shadowGeneralPreferencesActivity.getNextStartedActivity();
+        ShadowIntent shadowIntentGeneralPreferences = shadowOf(startedIntentGeneralPreferences);
+        assertEquals(PreferencesActivity.class.getName(),shadowIntentGeneralPreferences.getIntentClass().getName());
+
+        //Test for AdminPreferencesActivity
```

Comment:
I see. This one would be a piece of cake for `Espresso`, but for now, I think asserting `title` is fine.